diff -cr ../md5/md5.c md5/md5.c
*** ../md5/md5.c	2014-07-30 16:18:04.762077873 +0530
--- md5/md5.c	2014-07-24 17:43:32.022148503 +0530
***************
*** 55,65 ****
   */
  
  #include <sys/types.h>
! #include <sys/systm.h>
! #include <libkern/crypto/md5.h>
  
! #define	memset(x, y, z)	bzero(x, z);
! #define	memcpy(x, y, z)	bcopy(y, x, z)
  
  /*
   * The digest algorithm interprets the input message as a sequence of 32-bit
--- 55,65 ----
   */
  
  #include <sys/types.h>
! #include "md5.h"
! #include <string.h>
  
! //#define  memset(x, y, z)   bzero(x, z);
! //#define  memcpy(x, y, z)   bcopy(y, x, z)
  
  /*
   * The digest algorithm interprets the input message as a sequence of 32-bit
